index.js

```diff
@@ -0,0 +1,107 @@
+const express = require('express');
+const server = express();
+const knex = require('knex')
+const knexConfig = require('./knexfile');
+const cors = require('cors');
+const session = require('express-session');
+const bcrypt = require('bcryptjs');
+const db = knex(knexConfig.development);
+const KnexSessionStore = require('connect-session-knex')(session);
```

Comment:
Cool job setting up your own store with knex! Realize you don't have to in order to get this figured out, but as the express-session site says, you do want to set up a proper store like you're doing here